ledfx/effects/melt_and_sparkle.py

```diff
@@ -50,9 +50,9 @@ class HuxleyMelt(AudioReactiveEffect, HSVEffect):
             ): vol.All(vol.Coerce(float), vol.Range(min=0, max=1)),
             vol.Optional(
                 "strobe_width",
-                description="Percussive strobe width, in pixels",
-                default=10,
-            ): vol.All(vol.Coerce(int), vol.Range(min=0, max=1000)),
+                description="Percussive strobe width, from one pixel to max of 10% of the total length",
```

Comment:
I had what I thought were great sparkles on my setup with quite wide, can we go upto 30 or even 50%

Reply:
It looks like this may actually be 100% in the code, it's still early 😴

Reply:
Cool, i like that, so only description needs to catch up...